README.adoc

```diff
@@ -454,6 +455,48 @@ tomcat:
 <4> Optional parameter to configure sleeping time between log downloading. You can set in seconds using _s_ or miliseconds using _ms_. By default time unit is miliseconds and value 500.
 <5> Optional parameter to configure number of retries to be done. By default 10 iterations are done.
 
+Custom Await strategy:
+
+[source, java]
+.org.arquillian.cube.docker.impl.await.CustomAwaitStrategyImpl
+----
+public class CustomAwaitStrategyImpl implements AwaitStrategy {
+
+  Await params;
+  DockerClientExecutor dockerClientExecutor;
+  Cube<?> cube;
+
+  public void setCube(Cube<?> cube) {
```

Comment:
@bartoszmajsak Basically I need to pass both parameters because with `Cube` you can get some information about bindings. But I also pass the DockerClientExecutor because there is where all operations to Docker host are defined (even you can get `DockerClient` which is the base class from docker-java ) the reason is that in every release Docker is adding more and more operations so a user might need one and other user another one, if we make all operations through `Cube` means that we will need to keep update, and possibly some users will open issues because we add more operations on that class. If we find that some operations are repeated a lot, we can add it in `Cube` but at least we left an open door for a workaround (by using `DockerClient`)